zulip_bots/zulip_bots/bots/monkeytestit/doc.md

```diff
@@ -0,0 +1,57 @@
+# monkeytest.it bot
+
+## Quick overview
+
+This bot provides a quick way to check a site from a chat.
+Using user's monkeytestit's API  key, user can check a website
+for certain failures that user wanted to see.
+
+## Setting up API key
+
+1. Get your monkeytest.it API key, located in your
+   [dashboard](https://monkeytest.it/dashboard).
+
+2. Open `monkeytestit.conf` file. Create one if somehow it
```

Comment:
We actually don't want people to modify the file directly.  We want folks to create copies of the file in their home directory.

Reply:
Edited the step